vision_agent/agent/vision_agent_coder.py

```diff
@@ -674,6 +678,7 @@ def chat_with_workflow(
         chat: List[Message],
         test_multi_plan: bool = True,
         display_visualization: bool = False,
+        customized_tool_names: List[str] = [],
```

Comment:
I would make this default to `None`, there's a weird gotcha with mutable default arguments in python https://docs.python-guide.org/writing/gotchas/ and then inside the function you can check if it's `None` and set it to `[]`